ml/src/main/scala/org/apache/spark/ml/h2o/algos/H2OGBM.scala

```diff
@@ -78,11 +68,11 @@ trait H2OGBMParams extends H2OSharedTreeParams[GBMParameters] {
   //
   // Param definitions
   //
-  final val learnRate = doubleParam("learnRate")
-  final val learnRateAnnealing = doubleParam("learnRateAnnealing")
-  final val colSampleRate = doubleParam("colSampleRate")
-  final val maxAbsLeafnodePred = doubleParam("maxAbsLeafnodePred")
-  final val predNoiseBandwidth = doubleParam("predNoiseBandwidth")
+  private final val learnRate = doubleParam("learnRate")
+  private final val learnRateAnnealing = doubleParam("learnRateAnnealing")
```

Comment:
Do we need ```final``` if the field is ```private```. It can't be overridden anyway.

Reply:
good point, no need to make them final